platform-sdk/swirlds-common/src/main/java/com/swirlds/common/system/address/AddressBookUtils.java

```diff
@@ -44,9 +44,14 @@
  *     <li>memo field (optional)</li>
  * </ul>
  * Example: `address, 22, node22, node22, 1, 10.10.11.12, 5060, 212.25.36.123, 5060, memo for node 22`
+ * <p>
+ * The last line of the config.txt address book contains the nextNodeId value in the form of: `nextnodeid, 23`
```

Comment:
example needs to be updated from `nextnodeid` to `nextNodeId`

Reply:
updated